ui/src/api/questions.ts

```diff
@@ -13,6 +13,10 @@ const Questions = {
     const url = `${API_PREFIX}/${questionId}`
     return wrap(Axios.get(url))
   },
+  Edit(questionId: string, payload: object) {
```

Comment:
passing "object" as a type is not any better than using "any". The payload expects a number of particular fields like body, title and tags, those are easy to define explicitly